tasks/libs/copyright.py

```diff
@@ -54,6 +54,7 @@
     '^// This file includes software developed at CoreOS',
     '^// Copyright 2017 Kinvolk',
     '^// Copyright 2021 The Vitess Authors.',
+    '^// This file is licensed under the MIT License.',
```

Comment:
I'm slightly worried that this is a little generic and could appear in random opensource projects for which we wouldn't enforce the rules by mistake if we were to add this. Would it be possible to find a more targeted way to do this?

Reply:
AFAIK this is only used in the `lint-copyrights` task, which ensures that a copyright header is present in each `.go` file in the `datadog-agent` repository (it doesn't look at the code from other repositories we import, that is something the `lint-licenses` task does). This rule wouldn't change that behavior (what it says is that the MIT license header is a valid copyright header), so is there a specific risk here to adding this rule you're thinking of?

cc @sgnn7 who may have more context on what the `lint-copyrights` task should do in this case.

Reply:
You're right, I thought these would be also used on files from other projects, but they're not... In which case I think we're safe here and I can approve.

Reply:
@KSerrania / @bkabrda Just caught up with this comment and it's already merged but yes, this is used for checking if an approved copyright header is in all `.go` files in strictly just this repo so I think we're good.